test/StorageTests/RedisGrainStorageTests.cs

```diff
@@ -38,7 +35,10 @@ public class RedisGrainStorageTests
         [Fact]
         public async Task SubscribedGrainStillSubscribedAfterDeactivation()
         {
-            using (var clusterFixture = new ClusterFixture())
+            var clusterFixture = new ClusterFixture();
+            await clusterFixture.Start();
+            using (clusterFixture)
```

Comment:
Is this using the new C# using() that disposes at the end of the function or just assuming the next line is the implicit block executed? If the latter, I'd much prefer wrapping around with { } to avoid confusion/bugs with changes later.

Reply:
And elsewhere

Reply:
Done, I added in explicit scope for those usings.